webpack.prod.config.js

```diff
@@ -13,6 +15,17 @@ function resolvePrivateEnvConfig(filePath) {
 }
 resolvePrivateEnvConfig('.env.private');
 
+const plugins = []
+// PurgeCSS
+if (process.env.USE_PURGECSS) {
+  plugins.push(
+    new PurgeCSSPlugin({
+      paths: [glob.sync('node_modules/@edx/paragon/**/*', {nodir: true})],
```

Comment:
[clarification] Does this mean PurgeCSS will _only_ run for Paragon source files? Wouldn't we want to analyze all relevant content, including the micro-frontend's source code?

For example, https://github.com/openedx/frontend-app-ecommerce/pull/32/files

Also, https://github.com/openedx/frontend-app-profile/pull/440 (this PR removed PurgeCSS for the same reasons of it removing way too much css lol).